src/App.js

```diff
@@ -1,5 +1,10 @@
+import { useState } from 'react';
+import Animal from './components/Animal';
 const App = () => {
-  return <h1 className='text-3xl text-[#017880] font-bold flex justify-center'>React Intoductory Exercise!</h1>;
+  const [animals, setAnimals] = useState(['🐶 dog', '🐈 cat', '🐣 chicken', '🐄 cow', '🐑 sheep', '🐴 horse']);
+
+  const animalsElement = animals.map((animal, index) => <Animal key={index} animal={animal} />);
```

Comment:
One task for this exercise is to pass the list as a prop inside a custom component rather than creating a list inside the component.

Reply:
![image](https://user-images.githubusercontent.com/51261911/224680723-e606b0de-5f94-4690-a35e-9f8eaeea924a.png)
One of the task is to pass the list as prop as said in the screenshot above